qlty-check/src/planner/invocation_directory.rs

```diff
@@ -265,7 +265,9 @@ mod test {
 
         for (target, result) in targets_results {
             let invocation_directory = invocation_directory_planner.compute(&target).unwrap();
-            assert_eq!(invocation_directory, result);
+            let result_str = path_to_string(result);
+            let result = result_str.split("/./").last().unwrap_or(&result_str);
+            assert!(invocation_directory.ends_with(result));
```

Comment:
Is this change related to the error if prepare_script fails behavior?

Reply:
This is fixing a macOS temporal unit test failure.